src/components/activity/TimelineItemsPanel.svelte

```diff
@@ -17,54 +19,59 @@
 
 <Panel padBody={false}>
   <svelte:fragment slot="header">
-    <GridMenu {gridSection} title="Activity & Resource Types" />
+    <GridMenu {gridSection} title="Activity, Resource, Event Types" />
   </svelte:fragment>
 
   <svelte:fragment slot="body">
-    <Tabs class="activity-resource-tabs" tabListClassName="activity-resource-tabs-list">
+    <Tabs class="timeline-items-tabs" tabListClassName="timeline-items-tabs-list">
       <svelte:fragment slot="tab-list">
-        <Tab class="activity-resource-tab"><DirectiveAndSpanIcon /> Activities</Tab>
-        <Tab class="activity-resource-tab"><TimelineLineLayerIcon /> Resources</Tab>
+        <Tab class="timeline-items-tab"><DirectiveAndSpanIcon /> Activities</Tab>
+        <Tab class="timeline-items-tab"><TimelineLineLayerIcon /> Resources</Tab>
+        <Tab class="timeline-items-tab"><ExternalEventIcon /> Events</Tab>
       </svelte:fragment>
       <TabPanel>
         <ActivityList />
       </TabPanel>
       <TabPanel>
         <ResourceList />
       </TabPanel>
+      <TabPanel>
+        <ExternalEventTypeList />
+      </TabPanel>
     </Tabs>
   </svelte:fragment>>
 </Panel>
 
 <style>
-  :global(.tab-list.activity-resource-tabs-list) {
+  :global(.tab-list.timeline-items-tabs-list) {
     background-color: var(--st-gray-10);
+    overflow-x: scroll;
```

Comment:
This causes a permanent bar to appear below the tabs. I'd be in favor of just letting the entire panel scroll horizontally instead of just scrolling the tabs. In the future it'd be even better to have the tabs be more responsive and hide the text or icons if a set width threshold is hit.